packages/server/test/index.test.tsx

```diff
@@ -533,3 +536,36 @@ describe('createCaller()', () => {
     sub.start();
   });
 });
+
+// regression https://github.com/trpc/trpc/issues/527
+test('void mutation response', async () => {
+  const { client, close, wssPort, router } = routerToServerAndClient(
+    trpc
+      .router()
+      .mutation('undefined', {
```

Comment:
I wonder if we should have a test for `return undefined` as well? Should be the same, but I am not sure if `void` always equals `undefined`.

Reply:
`void` === `undefined`